python_backend/api/v1/router.py

```
@@ -0,0 +1,43 @@
+- FastAPI 애플리케이션의 API 버전 1 라우터 설정
+- 이 파일은 API 엔드포인트를 모듈화하여 관리하기 위한 라우터 설정 
+- 각 API 엔드포인트는 별도의 모듈로 분리되어 있으며, 이 파일에서 라우터를 포함
+
+라우터 관련 로직을 한 곳에 모으고 복수의 endpoint 파일을 router.py 에서 한번에 import 및 
+등록하게 만듦
+
+주의: 각각 파일 내부에 router= APIRouter() 를 선언해야 함
+"""
+
+from fastapi import APIRouter
+from api.v1.endpoints.health import router as health_router
```

Ruff (0.12.2)

`api.v1.endpoints.health.router` imported but unused

Remove unused import: `api.v1.endpoints.health.router`

(F401)